tests/unit/session/create.spec.ts

```diff
@@ -137,3 +141,98 @@ test.group('Register create', (group) => {
     })
   })
 })
+
+test.group('Login create', (group) => {
+  // Setup and teardown functions are called one time before and after
+  group.setup(async () => {
+    testUtils.db().truncate()
+    await User.create(user)
+  })
+  // clear user created after all tests
+  group.teardown(async () => testUtils.db().truncate())
+
+  test('ensure user can login', async ({ client }) => {
+    const response = await client.post('/login').json({
+      email: user.email,
+      password: user.password,
+    })
+
+    response.assertStatus(200)
+    response.assertBodyContains({
+      message: 'User logged in successfully',
+    })
+  })
+
+  test('ensure user cannot login with invalid email', async ({ client }) => {
+    const response = await client.post('/login').json({
+      email: 'test',
+      password: user.password,
+    })
+
+    response.assertStatus(422)
+    response.assertBodyContains({
+      errors: [
+        {
+          message: 'The value is not a valid email address',
+          rule: 'email',
+          field: 'email',
+        },
+      ],
+    })
+  })
+
+  test('ensure user cannot login with invalid password', async ({ client }) => {
+    const response = await client.post('/login').json({
+      email: user.email,
+      password: 'pass',
+    })
+
+    response.assertStatus(422)
+    response.assertBodyContains({
+      errors: [
+        {
+          message: 'The password must be at least 8 characters long',
```

Comment:
Not sure if the error message has to be "not long enough". I prefer "wrong password", no ?